### ISSUE TITLE
Best way to do with return in foreach?

### ISSUE THREAD

**Issue by roukmoute** [DusanKasan/Knapsack#31]
Hi!

I would like to know how can I use pipeline collection with this code:

```php
foreach ($myArrayOfAsciiCode as $ascii) {
    if (strpos($path, chr($ascii))) {
        return true;
    }
}

return false;
```

I do this:

```php
return !Collection::from(self::INVALID_PATH_ASCII)
                  ->filter(
                      function (string $ascii) use ($path): bool {
                          return strpos($path, chr($ascii));
                      }
                  )
                  ->isEmpty();
```

Is this the better way?

Thanks

**Comment by DusanKasan** [comment]
Hi. The [some](http://dusankasan.github.io/Knapsack/#Operations-some) operation does exactly what you need. Let me know if you have further questions.

**Comment by jdreesen** [comment]
You could use [`some`](https://dusankasan.github.io/Knapsack/#Operations-some) (the method names in the docs seem to be wrong, though. I'll create a PR to fix this):

```php
Collection::from($myArrayOfAsciiCode)->some(function ($ascii) use ($path) {
   return strpos($path, chr($ascii)) !== false;
});
```

**Comment by DusanKasan** [comment]
And I just realized the docs are showing wrong examples. Need to fix asap. Refer to tests for examples for now.

**Comment by jdreesen** [comment]
Fixed in #32

**Comment by roukmoute** [comment]
Thanks for your rapidity guys :blush:!
